### PR TITLE
Adding SSL.com CA

### PR DESCRIPTION
**Pull request by codelux**
SSL.com is a participant in the Microsoft Root CA program and is also referenced in other code signing related pages (see https://docs.microsoft.com/en-us/windows-hardware/drivers/dashboard/code-signing-cert-manage)

### CODE REVIEW

File: docs/create-packages/Sign-a-Package.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Signed packages allows for content integrity verification checks which provides
 
 ## Get a code signing certificate
 
-Valid certificates may be obtained from a public certificate authority such as [DigiCert](https://www.digicert.com/code-signing/), [Global Sign](https://www.globalsign.com/en/code-signing-certificate/), [Comodo](https://www.comodo.com/e-commerce/code-signing/code-signing-certificate.php), [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml), etc. The complete list of certification authorities trusted by Windows can be obtained from [http://aka.ms/trustcertpartners](/security/trusted-root/participants-list).
+Valid certificates may be obtained from a public certificate authority such as [SSL.com](https://www.ssl.com/certificates/code-signing/), [DigiCert](https://www.digicert.com/code-signing/), [Global Sign](https://www.globalsign.com/en/code-signing-certificate/), [Comodo](https://www.comodo.com/e-commerce/code-signing/code-signing-certificate.php), [Certum](https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml), etc. The complete list of certification authorities trusted by Windows can be obtained from [http://aka.ms/trustcertpartners](/security/trusted-root/participants-list).
```
Comment (JonDouglas) [inline docs/create-packages/Sign-a-Package.md:17]: Would you be willing to convert this section to a bullet list sorted alphabetically? We tend to keep that standard in other locations like https://docs.microsoft.com/en-us/nuget/hosting-packages/overview

**Review by JonDouglas — CHANGES_REQUESTED** [review]
Please convert this section to a alphabetical list to not introduce bias and be consistent with other NuGet docs.

**Comment by JonDouglas** [comment]
Closing this PR due to inactivity. Please re-open or comment if you'd like to revisit this.

File: docs/create-packages/Sign-a-Package.md (hunk 1 above)
Comment (codelux) [reply docs/create-packages/Sign-a-Package.md:17]: sure, will do

**Pull request closed without merging**